### ISSUE TITLE
Incorrect sample `pre-commit` config

### ISSUE THREAD

**Issue by jamesbraza** [jsh9/pydoclint#95]
From https://github.com/jsh9/pydoclint/blob/0.3.8/README.md#23-as-a-pre-commit-hook

```yaml
- repo: https://github.com/jsh9/pydoclint
  rev: <latest_tag>
  hooks:
    - id: pydoclint
      args:
        - [--style=google, --check-return-types=False]
```

The correct form of `args` should be either:

```yaml
      args: [--style=google, --check-return-types=False]
```

or:

```yaml
      args:
        - --style=google
        - --check-return-types=False
```

Thank you for making this tool btw!

**Comment by jsh9** [comment]
Thanks for finding this error!  I fixed it at: https://github.com/jsh9/pydoclint/commit/03c2cebee90eda1103d9aa8b0aa827f972b07d88
